client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -295,12 +296,13 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 		}
 
 		if ( EditorActions.GetGutenboardingStatus === action ) {
-			// TODO - In future iterations, replace window param with gutenboarding site info.
-			const urlParams = new URLSearchParams( window.location.search );
 			const isGutenboarding =
-				config.isEnabled( 'gutenboarding' ) && urlParams.has( 'is-gutenboarding' );
+				config.isEnabled( 'gutenboarding' ) &&
+				this.props.siteCreationFlow === 'gutenboarding' &&
+				this.props.isSiteUnlaunched;
 			ports[ 0 ].postMessage( {
 				isGutenboarding,
+				isSiteLaunched: ! this.props.isSiteUnlaunched,
```

Comment:
I don't think this PR uses the new `isSiteLaunched` message prop anywhere. Do you think it'll be used in the future?

Reply:
I left it in there by mistake! Thanks for the 👀